### DISCUSSION TITLE
Suggestion compressed audio data

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#734] [Q&A] [answered]
I record and play successfully i2s data 32 bit using wave encoder  then why  I an having the impression that encoder need 16 BiT data ? 

If i am facing problem in using SBC encoder in same above pattern then it means the problem is some were else its not 32 BiT issue , rights ? 

If issue is of 32 Bit then why wave encoder and decoder work successfully  ?

**Comment by pschatzmann** [comment, marked as answer]
WAV works completely different: it's actually not encoding at all, but just adding a header in the beginning of the PCM audio stream, so that the decoder knows how to interpret the data. The header contains the sample rate, channels and bits per sample.

SBC is uses a lossy data compression algorithm
